static/templates/stream_settings/stream_types.hbs

```diff
@@ -1,3 +1,15 @@
+{{#if ask_to_announce_stream}}
+    <div id="announce-new-stream">
+        <label class="checkbox">
+            <input type="checkbox" name="announce" value="announce" checked />
+            <span></span>
+            {{t "Announce new stream in #"}}{{notifications_stream}}
```

Comment:
I am not sure whether this is the correct way to tag this string for translation. You can see how we do this in existing cases like in deactivate stream modal in `static/templates/confirm_dialog/confirm_deactivate_stream.hbs` file. You can refer to https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations.

Reply:
Also, we want to have the stream name in `bold`, right? The issue description mentions the stream name in bold and we do highlight the stream names in some other cases as well.